fabric-item-groups-v0/src/main/java/net/fabricmc/fabric/api/client/itemgroup/FabricItemGroupBuilder.java

```diff
@@ -76,6 +77,16 @@ public FabricItemGroupBuilder stacksForDisplay(Consumer<List<ItemStack>> appende
 	 * @return a reference to the FabricItemGroupBuilder
 	 */
 	public FabricItemGroupBuilder appendItems(Consumer<List<ItemStack>> stacksForDisplay) {
```

Comment:
Would you mind stating that this bypasses items specifying a creative tab directly, and then explain how you can get this behavior by using the other function?